src/commands/sourcemaps/upload.rs

```diff
@@ -328,6 +340,17 @@ fn process_sources_from_paths(
             url_prefix = &url_prefix[..url_prefix.len() - 1];
         }
 
+        if inject_files {
+            let mut report = InjectReport::default();
+            for source in &sources {
+                if source.path.extension().map_or(false, |ext| ext == "js") {
```

Comment:
We could also have `.mjs` and `.cjs` files

Reply:
I suppose that means they should also be handled in the `sourcemaps inject` command?

Reply:
absolutely, yes

Reply:
What about `ts`, `tsx`, `jsx`, `cts` and `mts`? 👀 (they are all valid)

Reply:
Though not as production artifacts I would say. They are all being transpiled to one of `[cm]?js` before they are executed by either node or browsers.